curriculum/challenges/english/25-front-end-development/quiz-styling-forms/66ed9043f45ce3ece4053ebf.md

```diff
@@ -17,439 +17,443 @@ Answer all of the questions below correctly to pass the quiz.
 
 #### --answer--
 
-Placeholder answer
+It removes the browser's default appearance, allowing you to fully customize the design.
 
-### --question--
+ ### --question--
 
 #### --text--
 
-Placeholder question
+What is a common issue that arises when using `appearance: none` on a select element?
 
 #### --distractors--
 
-Placeholder distractor 1
+It makes the `select` element invisible.
 
 ---
 
-Placeholder distractor 2
+It disables the dropdown functionality.
 
 ---
 
-Placeholder distractor 3
+It breaks form validation.
 
 #### --answer--
 
-Placeholder answer
+It removes the dropdown arrow, requiring you to add a custom arrow or indicator.
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+What issue can occur when using HTML5 validation attributes like `required` without custom messaging?
 
 #### --distractors--
 
-Placeholder distractor 1
+It automatically validates all inputs without user interaction.
 
 ---
 
-Placeholder distractor 2
+It prevents the form from being submitted if all fields are filled.
 
 ---
 
-Placeholder distractor 3
+It allows users to see detailed error messages for each input.
 
 #### --answer--
 
-Placeholder answer
+Users may not understand why their submission fails, leading to frustration if no helpful feedback is provided.
+
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+What issue can arise when no margin is added between the header and the content?
 
 #### --distractors--
 
-Placeholder distractor 1
+It makes the content load faster.
 
 ---
 
-Placeholder distractor 2
+It improves the overall readability of the content.
 
 ---
 
-Placeholder distractor 3
+It automatically centers the content under the header.
 
 #### --answer--
 
-Placeholder answer
+The content may appear cramped or overlap with the header, making it difficult to read and reducing visual clarity.
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+What issue may arise with custom fonts when used in form elements?
 
 #### --distractors--
 
-Placeholder distractor 1
+They may not display correctly on all browsers.
 
 ---
 
-Placeholder distractor 2
+They always load faster than system fonts.
 
 ---
 
-Placeholder distractor 3
+They prevent the use of responsive design.
 
 #### --answer--
 
-Placeholder answer
+Custom fonts may not be accessible to users if they are not properly loaded or fallback fonts are not specified.
+
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+What issue can arise when using `overflow: hidden;` on form elements?
 
 #### --distractors--
 
-Placeholder distractor 1
+It improves the form’s overall accessibility.
 
 ---
 
-Placeholder distractor 2
+It automatically adjusts the form to fit within smaller screens.
 
 ---
 
-Placeholder distractor 3
+It ensures that all text content is always visible.
 
 #### --answer--
 
-Placeholder answer
+It can cause content, such as text or labels, to be cut off if the element is resized or if padding is used incorrectly.
+
```

Comment:
```suggestion
It can cause content, such as text or labels, to be cut off if the element is resized or if padding is used incorrectly.
```

Extra line break.

Reply:
Hi @naomi-lgbt,
Thank you for your feedback! I’ll make the necessary changes as soon as possible and commit them shortly.